mod/item.php

```diff
@@ -175,6 +175,19 @@ function item_post(&$a) {
 	$app         = ((x($_REQUEST,'source'))      ? strip_tags($_REQUEST['source'])  : '');
 	$extid       = ((x($_REQUEST,'extid'))       ? strip_tags($_REQUEST['extid'])  : '');
 
+	// Check for multiple posts with the same message id (when the post was created via API)
+	if (($message_id != '') AND ($profile_uid != 0)) {
```

Comment:
That's the opposite of me, I grew up in the world of C and I only started using literal `AND` and `OR` with SQL.

Now, I don't have a problem with using literal conditional operators in PHP, but as with quotes and indentation style, we still need to use the same everywhere in Friendica, at least the same in a single file, which is not currently the case.

See:
- https://github.com/friendica/friendica/blob/master/mod/item.php#L32
- https://github.com/friendica/friendica/blob/master/mod/item.php#L51
- https://github.com/friendica/friendica/blob/master/mod/item.php#L53
- you get it